src/AppBundle/Controller/RestaurantController.php

```diff
@@ -253,6 +257,16 @@ public function indexAction($id, $slug, Request $request, SlugifyInterface $slug
             }
         }
 
+        if ($restaurant->getState() === 'pledge') {
+
+            $numberOfVotes = count($restaurant->getPledge()->getVotes());
+
+            return $this->render('@App/admin/restaurant_pledge_accepted.html.twig', [
```

Comment:
The path of this Twig template is wrong, it is not an admin area template. 
It should be `@App/restaurant/pledge.html.twig`